hbase-client/src/main/java/org/apache/hadoop/hbase/client/HTable.java

```diff
@@ -476,20 +481,29 @@ public void batch(final List<? extends Row> actions, final Object[] results, int
   @Override
   public <R> void batchCallback(final List<? extends Row> actions, final Object[] results,
     final Batch.Callback<R> callback) throws IOException, InterruptedException {
-    doBatchWithCallback(actions, results, callback, connection, pool, tableName);
+    doBatchWithCallback(actions, results, callback, connection, pool, tableName, requestAttributes);
   }
 
   public static <R> void doBatchWithCallback(List<? extends Row> actions, Object[] results,
     Batch.Callback<R> callback, ClusterConnection connection, ExecutorService pool,
     TableName tableName) throws InterruptedIOException, RetriesExhaustedWithDetailsException {
+    doBatchWithCallback(actions, results, callback, connection, pool, tableName,
+      Collections.emptyMap());
```

Comment:
why not pass the requestAttributes here?

Reply:
oh wait, i see what you did here. Though since there are only two usages i think i'd prefer you not add an overload.

Reply:
because this is a public static method I think we need to continue to expose the existing signature and, without adding requestAttributes, we have nothing to pass in here

Reply:
>  think i'd prefer you not add an overload.

that sounds good to me — just to sanity check, is this okay in an `InterfaceStability.Stable` class?